packages/backend/src/managers/playground.ts

```diff
@@ -213,17 +213,22 @@ export class PlayGroundManager {
           if (!q) {
             throw new Error('query not found in state');
           }
+          q.error = undefined;
           q.response = result as ModelResponse;
           this.queries.set(query.id, q);
-          this.sendQueriesState().catch((err: unknown) => {
-            console.error('playground: unable to send the response to the frontend', err);
-          });
+          void this.sendQueriesState();
```

Comment:
Remove the void and add the catch

Reply:
May I ask why ? the void is to avoid waiting on it, so we do not block the thread without being forced to add a catch clause

What do you want to do in the catch block ? The method `sendQueriesState` is a calling the PostMessage api method, it _cannot_ fails as the api does not say error may occurs, so I don't see the risk of having `unhandledRejection` raised. 
The console.error in the catch block would in my opinion just add lines that are not useful nor relevant. We might want the advice of @benoitf on the matter

I think, that the sendQueriesState should not be `async`, and the implemention inside it should be responsible of `catching` unlikely issue

Reply:
adding the void makes the code less readable and potentially confusing for me and it does not add any value as it is just there to fake the compiler. You're calling the api so you cannot know if it returns or not an error. It may change.
IMO the goal is not to write less lines but to make them more understandable for everyone so we're less prone to errors when we work on it.

Reply:
to me, the risk to have floating promise is much more higher by silently ignoring possible errors
so I would advise to never add `void` keyword

Either the method handle by itself the asynchronous execution (we did that in the past for telemetry so it's no longer an async method) so you can call `this.sendQueriesState()` and linter will be happy.

Or if it's a promise you need to wait for the execution or catch the error.

Probably we need to tune the no-floating-promises to ensure it's not added by mistake.

```
"@typescript-eslint/no-floating-promises": [ "error", {"ignoreVoid": false } ]
```

Reply:
Related issue https://github.com/projectatomic/ai-studio/pull/197 I will rebase this PR when merged